add info about savepoints and updated checkpoints topic (#1707)
aiven/devportal: docs/products/flink/concepts/savepoints.rst

Savepoints
==========

> Savepoints in Aiven for Apache Flink® are snapshots of the current state of your Flink application. This allows you to gracefully shut down your Flink application and restart it later without losing any progress or state. 

Reviewer: ```suggestion
Savepoints in Aiven for Apache Flink® are snapshots of the current state of your Flink application. They are created when stopping an application deployment and allow you to restart it later on without losing any progress or state. 
```
